app-frontend/src/app/components/datasources/datasourceDeleteModal/datasourceDeleteModal.module.js

```diff
@@ -82,6 +95,14 @@ class DatasourceDeleteModalController {
             + 'Are you sure you wish to continue?</p>'
             + '</div>';
     }
+
+    displayPermissionMsg(acrCount) {
```

Comment:
I think this might work better as a tooltip on a disabled delete button. They shouldn't have to open the modal to know that they aren't able to delete the datasource.